pkg/jobs/hive/hive.go

```diff
@@ -32,6 +32,7 @@ import (
 
 const MCVUpgradeLabel = "cluster-curator-upgrade"
 const ForceUpgradeAnnotation = "cluster.open-cluster-management.io/upgrade-allow-not-recommended-versions"
+const UpgradeClusterversionBackoffLimit = "cluster.open-cluster-management.io/upgrade-clusterversion-backoffLimit"
```

Comment:
```suggestion
const UpgradeClusterversionBackoffLimit = "cluster.open-cluster-management.io/upgrade-clusterversion-backoff-limit"
```

I know we agreed on this naming but the capital `L` just looked so out of place and potentially error prone because annotation key might be case sensitive(?). Anyways could you please make this change? The rest looks good.

Reply:
Thanks @mikeshng ! Yes, the annotations are case sensitive. Let me change that.